pkg/codegen/python/gen.go

```diff
@@ -379,7 +379,7 @@ func (mod *modContext) genAwaitableType(w io.Writer, obj *schema.ObjectType) str
 	return awaitableName
 }
 
-func (mod *modContext) genResource(res *schema.Resource) (string, error) {
+func (mod *modContext) genResource(res *schema.Resource) (str string, err error) {
```

Comment:
Doesn't seem like this change is needed. The names aren't descriptive, and this isn't used to support a [naked return](https://tour.golang.org/basics/7)